docs/spec/dns-integration.md

```diff
@@ -81,11 +90,19 @@ func main() {
         ...
 }
 ``` 
-Make sure that your pod colocated with `nsm-coredns` and `nsm-coredns` has [environment variable](https://github.com/networkservicemesh/networkservicemesh/blob/master/docs/env.md) `USE_UPDATE_API=true`.
-See at example of usage `nsm-dns-monitor` in `test/applications/cmd/monitoring-dns-nsc`
+Make sure that your application pod includes the `nsm-coredns` and `nsm-coredns` containers and has [environment variable](https://github.com/networkservicemesh/networkservicemesh/blob/master/docs/env.md) `USE_UPDATE_API=true`.
+See an example of usage `nsm-dns-monitor` in `test/applications/cmd/monitoring-dns-nsc`
+
+### Using nsm-coredns and nsm-dns-monitor without changing the client's deployment configuration
+To inject the `nsm-coredns` and `nsm-dns-monitor` containers into a client's pod during deployment, you can simply deploy the [admission webhook](https://github.com/networkservicemesh/networkservicemesh/blob/master/docs/spec/admission.md). `Admission webhook` will automatically append the DNS specific containers to your `Network Service Client`.  When using the admission webhook there is no way to disable the insertion of these additional containers.
+
+## NSE Requirements
+In order for the application pod to try multiple DNS servers the NSEs must populate the DNScontext.   The SDK provides a means to populate the DNScontext based on environmental variables provided to the NSE container. These variables are: DNS_SEARCH_DOMAINS and DNS_SERVER_IPS.  The [icmp-responer](test/applications/cmd/icmp-responder-nse/main.go) has an example implementation.
+
```

Comment:
The icmp-responder is a great example of using the SDK... but the SDK out of the box isn't using those variables (though maybe we should consider that :) ).

Reply:
Yeah - As I responded above I need to be clear on what the SDK does vs. what the icmp-responder example does.